test-data/unit/check-plugin-attrs.test

```diff
@@ -2321,3 +2321,60 @@ reveal_type(b.x)  # N: Revealed type is "builtins.int"
 reveal_type(b.y)  # N: Revealed type is "builtins.int"
 
 [builtins fixtures/plugin_attrs.pyi]
+
+[case testDefaultHashability]
+from typing import Hashable
+from attrs import define
+
+@define
+class A:
+    a: int
+
+a: Hashable = A(1)
```

Comment:
I think that `reveal_type(A.__hash__)` is a better test.

Reply:
Changed!